Makefile

```diff
@@ -85,6 +85,9 @@ ifdef KEEP_ISO
     KEEP_ISO_FLAG = --keep-iso
 endif
 
+NUM_SECONDARY_MASTERS := $(or $(NUM_SECONDARY_MASTERS), 0)
```

Comment:
I must say i don't like this option. Not sure why it should be configurable at all

Reply:
Why not? Why can you configure regular masters and workers and not secondary (network) masters and workers?

Reply:
i don't see any usecase for it? so why to add irrelevant envS?